pkgs/development/libraries/sycl-info/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv
+, fetchFromGitHub
+, installShellFiles
+, cmake
+, ninja
+, ocl-icd
+, opencl-headers
+, lyra
+, nlohmann_json
+, ronn
+, doctest
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sycl-info";
+  version = "unstable-2019-11-19";
+
+  src = fetchFromGitHub {
+    owner = "codeplaysoftware";
+    repo = "sycl-info";
+    rev = "b47d498ee2d6b77ec21972de5882e8e12efecd6c";
+    sha256 = "0fy0y1rcfb11p3vijd8wym6xkaicav49pv2bv2l18rma929n1m1m";
+  };
+
+  buildInputs = [
+    nlohmann_json
+    ronn
+    opencl-headers
+    ocl-icd
+    doctest
+    lyra
+  ];
+
+  nativeBuildInputs = [
+    installShellFiles
```

Comment:
```suggestion
```
I don't see a targeted script. the executable is a binary

```
$ tree ./results/sycl-info
./results/sycl-info
├── bin
│   └── sycl-info
├── include
│   └── target_selector
│       ├── color_scope.hpp
│       ├── config.hpp
│       ├── export.hpp
│       └── target_selector.hpp
├── lib
│   ├── cmake
│   │   ├── sycl-info
│   │   │   ├── sycl-info-config.cmake
│   │   │   ├── sycl-info-config-version.cmake
│   │   │   ├── sycl-info-targets.cmake
│   │   │   └── sycl-info-targets-release.cmake
│   │   └── target-selector
│   │       ├── target-selector-config.cmake
│   │       ├── target-selector-config-version.cmake
│   │       ├── target-selector-targets.cmake
│   │       └── target-selector-targets-release.cmake
│   ├── libtarget-selector.so -> libtarget-selector.so.0
│   ├── libtarget-selector.so.0 -> libtarget-selector.so.0.1
│   └── libtarget-selector.so.0.1
└── share
    ├── doc
    │   └── sycl-info.1.html
    ├── licenses
    │   ├── sycl-info
    │   │   └── LICENSES.TXT
    │   └── target-selector
    │       └── LICENSES.TXT
    └── man
        └── man1
            └── sycl-info.1.gz
```

Reply:
Oops, fixed!